designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/scenariostatus/ScenarioStatusProvider.scala

```diff
@@ -78,6 +78,20 @@ class ScenarioStatusProvider(
     )
   }
 
+  def getActiveScenariosCountFor(processingType: ProcessingType)(implicit user: LoggedUser): Future[Int] = {
+    implicit val freshnessStatus: DataFreshnessPolicy = DataFreshnessPolicy.Fresh
+    deploymentStatusesProvider
```

Comment:
There is a little problem with this. The `EngineSideDeploymentStatusesProvider` doesn't take into consideration the little period when deployment is requested but, still not available on the engine side. It is why the name starts from `Engine`. It is why we have this:
```
    } else if (inProgressActionNames.contains(ScenarioActionName.Deploy)) {
      logStatusAndReturn(SimpleStateStatus.DuringDeploy)
```
check before we check status on the engine side in `getScenarioStatus` method.